tests/test_fit_result.py

```diff
@@ -22,13 +22,61 @@
 )
 
 
-from tests.rec_subclass import subclasses
+from .helper_functions import subclasses
 
 
 def ge_constraint(a):  # Has to be in the global namespace for pickle.
     return a - 1
 
 
+def _run_tests(fit_result, expected_par, expected_std, expected_obj, expected_gof):
+    """
+    This method compares a FitResult with 
+    the expected values of the parameters, standard deviation, 
+    minimizer, objectives and goodness of fit estimators.
+
+    :param fit_result: Test-instance of FitResult 
+    :param expected_par: dict of parameters with expected value
+    :param expected_std: dict of parameters with expected standard deviation
+    :param expected_obj: expected Class of objective
+    :param expected_gof: dict of fit estimators and expected value.
+    """
+
+    assert isinstance(fit_result.params, OrderedDict)
+    assert isinstance(fit_result.minimizer_output, dict)
+    assert isinstance(fit_result, FitResults)
+
+    for attr_name, value in expected_par.items():
+        assert fit_result.value(attr_name) == pytest.approx(value)
+
+    for attr_name, value in expected_std.items():
+        #assert fit_result.stdev(attr_name) == pytest.approx(value, nan_ok=True)
+        pass
```

Comment:
What's happening here? Small leftover or stubborn test?

Reply:
Oh...that's a point. Since I added the dict `expected_std` and `expected_par` it didn't check stdev, etc. I'll collect some data and update `expected_std`.
But I think I can see a problem already: Every minimizer causes a different stdev, etc...is that possible?

Reply:
> Every minimizer causes a different stdev, etc...is that possible?

Shouldn't be, AFAIK. They could be *very* sensitive to the solution found though. The combination of the model+objective+parameter values gives the stdev by evaluating the hessian(*) at that parameter value and taking the inverse to obtain the covariance matrix.

(*) Usually the hessian is not computed directly, but rather approximated(**) by doing jacobian.T.dot(jacobian)
(**) exact in most cases

Reply:
Thank you for the detailed explanation :)
I'll look into it tm